app/models/user.rb

```diff
@@ -287,7 +287,7 @@ def inactive_message
     I18n.t(:inactive_html, path_to_contact: '/contact').html_safe
   end
 
-  def avatar_url(size=nil,avatar_kind=nil)
+  def avatar_url(size=nil, kind=nil)
```

Comment:
That sounds great @gdpelican. 

I think kind should be removed from the arguments of the method. It seems like it's breaking a principal or two exposing the detail like that. 

I think I would classify this problem as resolved when there is a method that you can call without needing to pass any arguments, and it returns a usefully sized image, even if the user has not selected a profile image. IE: It should return some default image if there is no profile image.

Can I propose we use the following method signatures?

profile_image_url(size = :medium), which always returns a working image url.

and

has_profile_image? which returns `false` if profile_image_url will return the default image.

I think at this stage it's worth changing the name of the method and removing the buggy method, so that we can clean up the codebase properly.
 
Also.. it's considered funky and fresh, when accessing a hash with a default, to use #fetch. EG: IMAGE_SIZES.fetch(size, :medium).